org.scala-ide.sdt.core.tests/src/org/scalaide/core/text/DocumentTest.scala

```diff
@@ -216,4 +216,46 @@ class DocumentTest {
   @Test
   def lastOpt_on_empty_file_returns_none() =
     document("").lastOpt === None
+
+  @Test
+  def lineInformation_on_non_empty_file_succeeds() = {
+    val d = document("""|first
+                        |second
+                        |third
+                        |""".stripMargin)
+    d.lineInformation(1).text(d) === "second"
+  }
+
```

Comment:
JUnit tests can only have `Unit` as return type therefore it doesn't really matter. But I can add it.